app/views/public_articles/show.html.haml

```diff
@@ -1,3 +1,10 @@
+%meta{:name => "citation_title", :content => @article.title}
+- @article.authors_metadata.each do |author|
+  %meta{:name => "citation_author", :content => author}
+%meta{:name => "citation_publication_date", :content => @article.year}
+%meta{:name => "citation_journal_title", :content => "Rocznik Kognitywistyczny"}
+%meta{:name => "citation_issue", :content => @article.issue_title}
+
```

Comment:
Wydaje mi się, że te dane powinny być w sekcji header. Proszę poczytać o helperze `content_for` i zaimplementować to z jego użyciem (będzie potrzebna zmiana layoutu).